.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/)
+- [ ] I viewed the changes on the dev server (`yarn dev`) to confirm they render without error
```

Comment:
We could probably condense these by asking the reader to look at `/docs/contributing/documentation`

Reply:
Initially, I was thinking of a way to use a reference to the contribution guide to consolidate this and the previous item. However, I also think it would make sense to emphasize that the reader should view the changes on the dev server.

Do you see another other ways we can reduce the length of the checklist? Without the spell checking and Core concepts items, the checklist for docs changes has four items (which I think is okay) and the list for new docs pages has seven. Is there a way to trim the list for new docs pages to five items?

Reply:
I've done some additional consolidation.